src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -479,75 +527,33 @@ public void traverse() {
 
         final int sampleCount = sampleNameToVcfPath.size();
         final int updatedBatchSize = (batchSize == DEFAULT_ZERO_BATCH_SIZE) ? sampleCount : batchSize;
-        final int totalBatchCount = (sampleCount/updatedBatchSize) + (sampleCount%updatedBatchSize==0 ? 0 : 1);
-
+//        final int totalBatchCount = (sampleCount/updatedBatchSize) + (sampleCount%updatedBatchSize==0 ? 0 : 1);
+        final ImportConfig importConfig = createImportConfig(updatedBatchSize);
+        //TODO: Fix this.
+//        BiConsumer<Map<String, FeatureReader<VariantContext>>, Integer> closeReaders = (readers, batchCount) -> {
+//            progressMeter.update(intervals.get(0));
+//            logger.info("Done importing batch " + batchCount + "/" + totalBatchCount);
```

Comment:
Can you explain what this TODO is? Are the readers getting closed properly? (I notice that you deleted the `closeReaders()` method below...)

Reply:
Spurious comment - yes, the readers are being closed inside GenomicsDBImporter